W-14036077-add-note-event-tracking-log (#688)
mulesoft/docs-runtime-manager: runtime-manager/modules/ROOT/pages/event-tracking.adoc

    ]
  }
----

[NOTE]

> The event tracking log file (_$MULE_HOME/logs/events-%d{yyyy-dd-MM}-%i.log_) is not automatically deleted. You need to manually delete the logs periodically.

Reviewer: ```suggestion
The event tracking log file (`$MULE_HOME/logs/events-%d{yyyy-dd-MM}-%i.log`) is not automatically deleted. You must manually delete the logs periodically.
```
